jenkins-centos7-atomicapp/artifacts/kubernetes/jenkins-rc.json

```diff
@@ -0,0 +1,47 @@
+{
+  "kind": "ReplicationController",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "jenkins"
+  },
+  "spec": {
+    "replicas": "$replicas",
+    "selector": {
+      "app": "jenkins"
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "app": "jenkins"
+        },
+        "name": "jenkins"
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "jenkins",
+            "image": "$image",
+            "ports": [
+              {
+                "name": "jenkins-http",
+                "containerPort": 8080
+              }
+            ],
+            "volumeMounts": [
+              {
+                "mountPath": "/var/lib/jenkins",
+                "name": "jenkins-persistent-storage"
+              }
+            ]
+          }
+        ],
+        "volumes": [
+          {
+            "emptyDir": {},
```

Comment:
maybe worth adding a comment that data is lost after container terminated, persistent-storage suggest something different?! or am I wrong? hmm... is it simply bound to the host where to pod ran on initially?